src/Storage/custom/hybrid-2019-03-01/Update-AzStorageAccount.ps1

```diff
@@ -0,0 +1,373 @@
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        # The credentials, account, tenant, and subscription used for communication with Azure.
+        ${DefaultProfile},
+    
+        [Parameter(DontShow, HelpMessage='Wait for .NET debugger to attach')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Wait for .NET debugger to attach
+        ${Break},
+    
+        [Parameter(DontShow, HelpMessage='SendAsync Pipeline Steps to be appended to the front of the pipeline')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline
+        ${HttpPipelineAppend},
+    
+        [Parameter(DontShow, HelpMessage='SendAsync Pipeline Steps to be prepended to the front of the pipeline')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+        ${HttpPipelinePrepend},
+    
+        [Parameter(DontShow, HelpMessage='The URI for the proxy server to use')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use
+        ${Proxy},
+    
+        [Parameter(DontShow, HelpMessage='Credentials for a proxy server to use for the remote call')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call
+        ${ProxyCredential},
+    
+        [Parameter(DontShow, HelpMessage='Use the default credentials for the proxy')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy
+        ${ProxyUseDefaultCredentials}
+    )
+    
+    process {
+        if ($PSBoundParameters.ContainsKey("StorageEncryption")) {
+            $null = $PSBoundParameters.Add("EncryptionKeySource", "Microsoft.Storage")
+            $null = $PSBoundParameters.Remove("StorageEncryption")
+        }
+        if ($PSBoundParameters.ContainsKey("KeyVaultEncryption")) {
+            $null = $PSBoundParameters.Add("EncryptionKeySource", "Microsoft.Keyvault")
+            $null = $PSBoundParameters.Remove("KeyVaultEncryption")
+        }
+        if ($PSBoundParameters.ContainsKey("NoEncryption")) {
```

Comment:
A few changes are needed here:

(1) Need to pass the `DontShow` parameters into the Get cmdlet
(2) Cannot simply pass through the AsJob parameter.  In this case, we need to use Start-Job amd execute the rest of the parameters (without AsJob) synchronously inside the job